tests/qa_automation/qa_run.pm

```diff
@@ -209,6 +214,9 @@ sub run() {
     my $junit_type = $self->junit_type();
     assert_script_run("/usr/share/qa/qaset/bin/junit_xml_gen.py -n '$junit_type' -d -o /tmp/junit.xml /var/log/qaset");
     parse_junit_log("/tmp/junit.xml");
+
+    # Switch back to root-console
+    select_console('root-console');
```

Comment:
am I missing something or this step is relevant only for case when it is 'VIRTIO_CONSOLE' ?

in this case please add if (get_var('VIRTIO_CONSOLE')) so we won't waste time on selecting something which is already selected